lib/nrf9160_time/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NRF9160_TIME
+	bool "nRF9160 time library"
```

Comment:
Any chance you could give this a more generic name so we don't need to rename when/if another chip arises that supports this library, but is not named nrf9160 ?

Reply:
Yes, I was thinking ``date_time``could be a fitting name. Does the tech writers have any opinion here? @umapraseeda

Reply:
'Date-Time Library' looks good to me. What about something like 'Timestamp library' ? ... not sure if that is feasible.

Reply:
I like ["civil time"](https://en.wikipedia.org/wiki/Civil_time) to indicate that something supports time that's compatible with calendars and UTC.  Since this is primarily (AFAICT) guided to tracking UTC, rather than manipulating dates and times.

Reply:
This has not been resolved.